composer.json

```
@@ -54,7 +54,7 @@
         "payum/omnipay-v3-bridge": "^1.0",
         "omnipay/dummy": "^3.0",
         "omnipay/common": "^3.0",
-        "doctrine/dbal": "^2",
+        "doctrine/dbal": "^2.0 || ^3.0",
```

```suggestion
        "doctrine/dbal": "^2.7 || ^3.0",
```
Otherwise with low deps, it will fail.